### ISSUE TITLE
Appeal for inserting real map as semi-transparent background to facilitate drawing

### ISSUE THREAD

**Issue by Monolitho** [railmapgen/rmp#811]
I hope that we can upload images of real map and set the transparecy as the background of the project so that we can draw lines accordingly.

**Comment by Monolitho** [comment]
At least for now I have came up with an alternative plan: In Ubuntu, open rmp in a browser and set it to semitransparent via xprop(search how to use it in Google) After that you may open your image at the buttom of the browser for your reference.

**Comment by thekingofcity** [comment]
Hi, sorry for the late reply. As you may know, the new 5.0 release includes a master node that allows you to import any SVG content in the canvas. Thus, another workaround will be to import the map as SVG type and set this master node under others.

https://www.bilibili.com/video/BV1zXaUeCEx1/

**Comment by Monolitho** [comment]
Alright, but the version on the website is still 3.x.x... When can we use 5.0 release?

**Comment by 28yfang** [comment]
Hi, to use the 5.0 version please go to https://railmapgen.org

**Comment by Monolitho** [comment]
would you support other image formats in the future?

**Comment by thekingofcity** [comment]
Yes, there is a plan to support converting any raster images (e.g. jpg, png) to SVG and then inserting it. We currently do not have a timetable for this but should be available in the following year :)

**Comment by Monolitho** [comment]
Thanks a lot!
